src/util/message.h

```diff
@@ -75,8 +75,12 @@ struct messaget
         return false;
       if (config.options.get_bool_option("log-message"))
       {
-        std::time_t currentTime = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
-        std::string timeStr = (std::ostringstream{} << std::put_time(std::localtime(&currentTime), "%Y-%m-%d %H:%M:%S")).str();
+        std::time_t currentTime = std::chrono::system_clock::to_time_t(
+          std::chrono::system_clock::now());
+        std::string timeStr =
+          (std::ostringstream{}
+           << std::put_time(std::localtime(&currentTime), "%Y-%m-%d %H:%M:%S"))
+            .str();
         fmt::print(f, "[{}] File: {}, Line: {} ", timeStr, file, line);
```

Comment:
@Chriszai: According to @mikhailramalho's comments, should you move `timeStr` to the beginning, or did I miss something here?